main.go

```diff
@@ -4,7 +4,11 @@ import (
 	"context"
 	"flag"
```

Comment:
** Importing `"log"` before other imports is standard practice. Retain consistency in import order.